### PR TITLE
Csv download

### PR DESCRIPTION
**Pull request by reginafcompton**
@derekeder This pull request implements a CSV download for CUAMP. 

I essentially built out the CSV-data-creation with a system of FOR loops. (I played around with this library, but without much luck: https://github.com/evanplaice/jquery-csv) 

Does this look reasonable? or can you imagine a more standard way to do this?

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -28,6 +28,7 @@
   <script type="text/javascript" src="js/lib/bootstrap.min.js"></script>
   <script type="text/javascript" src="js/lib/cartodb.js"></script>
   <script type="text/javascript" src="js/lib/leaflet-google.js"></script>
+  <script type="text/javascript" src="js/lib/jquery.csv.min.js"></script>
```
Comment (derekeder) [inline _layouts/default.html]: do we need this if you're implementing manually?

File: js/utils.js [hunk 2]
```diff
@@ -25,3 +38,145 @@ var convertBoolean =  function convertBoolean(text) {
   else {
   }
 };
+
```
Comment (derekeder) [inline js/utils.js:41]: looks like you did some refactoring here. it would be better to do that in a separate PR since this one pertains just to CSV downloads

**Review by derekeder — COMMENTED** [review]
I tested this out, and it works as expected.

What kinds of issues did you run into using the jquery.csv library? from the look of it, its more for parsing CSVs than it is for downloading them, so maybe it wasn't the right fit.

**Comment by reginafcompton** [comment]
@derekeder all right - I removed the unnecessary dependency on jquery-csv. This library supposedly has array-to-csv functionality, but I could not get it to do to anything (apparently, this part of the library is under development...the library itself does not seem very well-maintained, in general). 

If this looks okay, then I'll merge, deploy, and send an email to Ben.
